config/install/tfa.settings.yml

```diff
@@ -1,6 +1,10 @@
 enabled: 0
+secret_key: ''
```

Comment:
We might want to avoid storing this in settings.

Please file an issue to integrate with the Key module (which I developed and can elaborate)